### ISSUE TITLE
Expanded Buttons Text Alignment Changes on Wrap

### ISSUE THREAD

**Issue by jamesstoneco** [foundation/foundation-emails#301]
**How can we reproduce this bug?**

Create an expanded button with a short amount of text.

**Write out the HTML (or Inky code) that causes the issue.**

Issue is the same with Inky or HTML.

``` html
<button href="#" class="expanded">short</button>
```

vs.

``` html
<button href="#" class="secondary expanded">some longer text causing it to wrap</button>
```

**What did you expect to happen?**

I expected both to be centered as they would be on F4Sites.

**What happened instead?**

Instead the alignment changed from _left_ on shorter text to _centered_ on longer text.

**What email clients does this happen in?**

I did not test on an email client but this appears on Chrome and FF Mac using Panini Server running from the command line.

**Comment by rafibomb** [comment]
visual 
![image](https://cloud.githubusercontent.com/assets/5652085/14118899/8912f15e-f5a0-11e5-9029-27cdf8909bc8.png)

**Comment by rafibomb** [comment]
This seems related to https://github.com/zurb/foundation-emails/issues/330

**Comment by rafibomb** [comment]
Resolved! 2.1 will be out next week!
